VideoLocker/src/main/java/org/edx/mobile/view/custom/URLInterceptorWebViewClient.java

```diff
@@ -16,33 +17,28 @@
 
 import org.edx.mobile.logger.Logger;
 import org.edx.mobile.util.BrowserUtil;
-import org.edx.mobile.util.StandardCharsets;
 import org.edx.mobile.util.Config;
 import org.edx.mobile.util.ConfigUtil;
 import org.edx.mobile.util.NetworkUtil;
+import org.edx.mobile.util.StandardCharsets;
+import org.edx.mobile.util.links.EdxCourseInfoLink;
+import org.edx.mobile.util.links.EdxEnrollLink;
 
 import roboguice.RoboGuice;
 
 /**
  * Created by rohan on 2/2/15.
- *
+ * <p/>
```

Comment:
Oh, what it's *for* is to insert a line break between the paragraphs when viewing the JavaDoc